.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```

Comment:
These example steps are clearly made for browser apps, maybe we should alter them like this:

**Docmaker config**
```yaml
layout: layout.html
data:
  - data.yaml
  - dynamic_data.js
pages:
  - "_titlepage.html"
  - "_toc.md"
  - "[0-9]+*.md"
assets:
  - styles.css
```

**To Reproduce**
1. Run `docmaker`
2. ...